### DISCUSSION TITLE
Allow deleting translations from translations interface

### DISCUSSION THREAD

**Discussion by badrange** [directus/directus#19895] [Feature Requests]
### Summary

When a person inserting content accidentally inserts content in the wrong language Directus automatically creates a translation for it and there is no way to "undo" this.

One way is to open a split view and copy all the fields to the right language, but then they would also need to be able to delete the translation with wrong language which they cannot do at the moment. 

I know it's possible to delete them from the translations collection but making it visible and telling editors to go find and delete their translation there isn't really an option as it'd be way too error prone.

Right now our workflow is that when an editor inserts content in the wrong language they will just archive that content completely and insert everything again in the correct language. Would be great if the translation UI had some kind of button/icon or whatever to delete the translation. That would save a lot of time.

### Motivation

We want Directus UI to be intuitive to content editors and less prone to allow them to make mistakes requiring extra work

### Detailed Design

Add a "delete translation" button to the Translation UI

### Must Have:
- Ability to delete translations created by mistake

### Should Have:
- Ability to delete translations in general

### Drawbacks

Any extra button has it's own cost in terms of complexity.

### Alternatives

Two options: 

1. When an editor inserts content in the wrong language they will just archive that content completely and insert everything again in the correct language. Would be great if the translation UI had some kind of button/icon or whatever to delete the translation. That would save a lot of time.
2. Delete them from the translations collection, but making it visible and telling editors to go find and delete their translation there isn't really an option as it'd be way too error prone.

### Adoption Strategy

This is not a breaking change. It could be added with some setting on the content collection that is by default on or off.

**Comment by taco-chen** [comment]
It is extremely important and lovely to have this feature!
